### PR TITLE
Add time, date, NTP tutorial

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: docs/tutorials/index.mdx [hunk 1]
```diff
@@ -279,6 +282,25 @@ Connect to a BLE device.
 
 </Boxes>
 
+## Common tasks
+<Boxes>
+<Box title="Date, time and NTP" to="misc/date-time">
```
Comment (kasperl) [inline docs/tutorials/index.mdx:287]:
```suggestion
<Box title="Date, time, and NTP" to="misc/date-time">
```

File: docs/tutorials/misc/date-time.mdx [hunk 2]
```diff
@@ -0,0 +1,160 @@
+# Date, time and NTP
```
Comment (kasperl) [inline docs/tutorials/misc/date-time.mdx:1]:
```suggestion
# Date, time, and NTP
```

File: tutorial_code/time/time_v1.toit [hunk 3]
```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2023 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the EXAMPLES_LICENSE file.
```
Comment (kasperl) [inline tutorial_code/time/time_v1.toit:3]: Didn't you change the name of the license file?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: docs/tutorials/index.mdx (hunk 1 above)
Comment (floitsch) [reply docs/tutorials/index.mdx]: done.

File: docs/tutorials/misc/date-time.mdx (hunk 2 above)
Comment (floitsch) [reply docs/tutorials/misc/date-time.mdx]: done.

File: tutorial_code/time/time_v1.toit (hunk 3 above)
Comment (floitsch) [reply tutorial_code/time/time_v1.toit]: Yes. In a parallel PR.
I changed this one and the other ones I missed.
done.

**Pull request merged**